packages/server/src/main/java/software/uncharted/terarium/hmiserver/models/dataservice/code/Code.java

```diff
@@ -46,7 +47,7 @@ public class Code extends TerariumAsset {
 	/* The optional metadata for this code */
 	@TSOptional
 	@Schema(accessMode = Schema.AccessMode.READ_ONLY, defaultValue = "{}")
-	@ElementCollection
```

Comment:
Code seemed to have originally worked - will this require a migration to support or will it just work?